doc/design/zarr-support.md

```diff
@@ -0,0 +1,17 @@
+The Zarr format is being used by NGFF to support the next generation of microscopy data. This involves a potentially 
+nested folder structure with named files inside the folder. In the DANDI this would mean:
+
+1. An asset is associated with a single zarr folder. From a user perspective this is still a single asset and the UI 
+   should not try to delve into the structure of the folder. The CLI should be able to download the entire tree. Matt
+   at kitware is looking into IPFS + NGFF, so we should at least keep that in mind.
```

Comment:
note: on DataLad end I might make each .zarr into a dedicated dataset. There are cons in that (e.g. no shared keystore between different .zarr files sharing some data) but it is the only way I see it to be done in a scalable fashion